UPGRADE.md

```diff
@@ -36,6 +36,294 @@ starts new workers, which take over from old workers before those old workers
 are terminated. In this way, Kong will serve new requests via the new
 configuration, without dropping existing in-flight connections.
 
+## Upgrade to `0.11.x`
+
+Along with the usual database migrations shipped with our major releases, this
+particular release introduces a few changes in behavior and, most of all, the
```

Comment:
`in behavior` is fine in this case

Reasoning: Since the text does not immediately follow with what behavior is being changed directly, otherwise you would use `in the behavior`.